pkg/crd/cloudprivateipconfig/v1/register.go

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName          = "network.openshift.io"
```

Comment:
This group is already in use - you'll need to pick a new one.

Reply:
ack will change